.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+	"presets": [ "@babel/preset-env" ]
```

Comment:
I had to look this up, and it may confuse others: this babel (js) has nothing to do with the Python babel we use for i18n here: https://github.com/internetarchive/openlibrary/blob/b0c1152a1785f2cd7c2e52565a6abf00256e33d5/openlibrary/i18n/__init__.py